### PR TITLE
Indev rewrite - Docs Update

### PR DESCRIPTION
**Pull request by LewisHogan**
Updated GuildChannel.type documentation.

(Now has type: String)

### CODE REVIEW

**Comment by acdenisSK** [comment]
You should fix those merge conflicts if you want this to be merged 👀

File: src/structures/GuildChannel.js [hunk 1]
```diff
@@ -35,8 +35,9 @@ class GuildChannel extends Channel {
      * The type of the Guild Channel
      * @type {String}
      */
-    //Returns either text, voice, or the id of the channel
-    this.type = data.type == 0 ? 'text' : data.type == 2 ? 'voice' : data.type.toString();
+    if(data.type === 0) this.type = 'text';
```
Comment (GusCaplan) [inline src/structures/GuildChannel.js]: i hope this change is never merged

Comment (LewisHogan) [inline src/structures/GuildChannel.js]: Can you be more specific about which part of the code (execution/concept) is horrific to you and what result you would prefer?

**Comment by hydrabolt** [comment]
Already implemented into Channel base class

**Pull request closed without merging**